libdevice/fallback-cmath.cpp

```diff
@@ -15,6 +15,16 @@
 // or remove any item in this file.
 // TODO: generate the DeviceLibFuncMap in sycl-post-link.cpp automatically
 // during the build based on libdevice to avoid manually sync.
+
+DEVICE_EXTERN_C
```

Comment:
These functions should be defined in the `wrapper` part and just call `__devicelib` versions, which should be defined here.

Reply:
Oh，sorry to miss the wrapper obj change，I will update soon.
Thanks.